front/document.send.php

```diff
@@ -38,7 +38,8 @@
 include('../inc/includes.php');
 
 if (!$CFG_GLPI["use_public_faq"]) {
-    Session::checkLoginUser();
+    // Might be displaying image for anonymous users on formcreator
+    Session::checkValidSessionId(true);
```

Comment:
I probably miss something here.

`front/document.send.php` endpoint can be called using 2 distinct params.
 - Using the `file` param, only dumps, user pictures and inventory files can be served. Are you using this case ?
 - Using the `docid` param, `Document::canViewFile()` method will be used to validate that user has right to see the file. If there is no rights in session, will this method be able to return `true` ?

Reply:
Full URL: `document.send.php?docid=11&itemtype=PluginFormcreatorQuestion&items_id=7`

I guess it falls under this condition:

![image](https://user-images.githubusercontent.com/42734840/211295448-145750eb-c391-4b49-9f40-a7f3b68cf413.png)

Which leads to `PluginFormcreatorForm ::canView()`, which is always true:

![image](https://user-images.githubusercontent.com/42734840/211295902-50a0d16e-fae4-4109-a0a9-d4896bc8525a.png)